api/src/components/coauthor/service.ts

```diff
@@ -142,10 +142,11 @@ export const updateConfirmation = async (publicationId: string, userId: string,
     return updateCoAuthor;
 };
 
-export const resetCoAuthors = async (publicationId: string) => {
+export const resetCoAuthors = async (publicationId: string, correspondingAuthor: string) => {
```

Comment:
this service shouldn't ask for the creator, it can process everything by the publicationId

Reply:
It needs to corresponding author, otherwise set their automatic approval to false.

Reply:
I know, but we can get the corresponding author by the publicationId